server/runtime/src/main/java/org/infinispan/server/Bootstrap.java

```diff
@@ -100,6 +100,13 @@ protected void handleArgumentCommand(String command, String parameter, Iterator<
             // Fall through
          case "--port-offset":
             properties.setProperty(Server.INFINISPAN_PORT_OFFSET, parameter);
+            int offset = Integer.parseInt(parameter);
+            if (!properties.containsKey(Server.JGROUPS_BIND_PORT)) {
+               properties.setProperty(Server.JGROUPS_BIND_PORT, Integer.toString(Server.DEFAULT_JGROUPS_BIND_PORT + offset));
+            }
+            if (!properties.containsKey(Server.JGROUPS_MCAST_PORT)) {
+               properties.setProperty(Server.JGROUPS_MCAST_PORT, Integer.toString(Server.DEFAULT_JGROUPS_MCAST_PORT + offset));
```

Comment:
this is going to change the UDP & MPING ports. 
Nodes aren't able to see each other anymore without manually setting this property.

Running these commands in 2 different terminals no longer form a cluster
```
$ ./bin/server.sh 
$ ./bin/server.sh -s server2 -o 100
```